### ISSUE TITLE
login not ok

### ISSUE THREAD

**Issue by JulianHH** [erdmannp/ZattooCmd#1]
hi :)
I added the login data but it says: login not okay everytime.
did the API change maybe?

cheers!

**Comment by svnt** [comment]
run into the same issue.

Fix:   

change line 15 in zapisession.py to

	ZAPI_URL = 'https://zattoo.com'


Btw, Any idea how to get the EPG for a specific channel?

**Comment by derdav3** [comment]
yeah unfortunately this doesn't work anymore, even with the right zapi url

**Comment by svnt** [comment]
works here OK here.
wrong country or login credentials? 
any error MSG?

**Comment by derdav3** [comment]
I'm in CH - also using a CH VPN doesn't help. Login credentials work - copy pasted to the site, and works.
output is only "Login not ok..." 
what would you need printed, to debug it?

**Comment by derdav3** [comment]
ok my bad - it couldnt read the cfg file... now the channels get loaded but after "-watch" i get an empty output

**Comment by derdav3** [comment]
coming back to this... again login not ok :/

**Comment by ventrixcode** [comment]
I think something is outdated
